scripts/validate_config.py

```diff
@@ -1,17 +1,38 @@
 """Usage: python scripts/validate_config.py"""
 from dynaconf.validator import ValidationError
 
-from robottelo.config import settings
+try:
+    from robottelo.config import settings
+except Exception as err:
+    print(f"Encountered the following exception, continuing for the sake of validation\n {err}")
+
+    from dynaconf import LazySettings
+    from robottelo.config.validators import VALIDATORS
+
+    settings = LazySettings(
```

Comment:
Not liking that this is duplicating the code in the config module itself, and then becomes sensitive to any changes that we make there.

I think its worth taking this a step farther, and creating a function in `robottelo/config/__init__.py` to return the raw settings object before validation.

Then `robottelo/config/__init__.py` can still create a module label `settings` and validate it, so existing imports don't change.  This validation script can use the function to get the same LazySettings instance as the rest of the framework, just without being validated before import.

Reply:
We would not. Within `robottelo.config` we would still define `settings`, just by calling the function that's been defined. No imports would change.

Reply:
Ping @spusateri !

Reply:
@mshriver @jyejare I think the most recent commit reflects the requested changes

Reply:
Actually, I'm not sure my solution is correct, does `get_settings` need to return the settings object _before_ they are validated in `robotello/config/__init__.py` or is it okay as-is?

Reply:
@spusateri I think so, else you will be in the loop!